build/dependencies.props

```diff
@@ -18,8 +18,8 @@
   <!-- Partner Versions-->
   <PropertyGroup>
     <CodeAnalysisVersion>1.3.0</CodeAnalysisVersion>
-    <MicrosoftBuildVersion>15.1.548</MicrosoftBuildVersion>
-    <NewtonSoftVersion>9.0.1</NewtonSoftVersion>
+    <MicrosoftBuildVersion>15.2.0-preview-000047-02</MicrosoftBuildVersion>
+    <NewtonSoftVersion>10.0.1</NewtonSoftVersion>
```

Comment:
nit: Could you change this name to the following to keep it consistent across repos (also when searching for next version change even this repo would show up): https://github.com/aspnet/Mvc/blob/dev/build/dependencies.props#L8